src/main/java/scrolls/elder/model/person/Person.java

```diff
@@ -29,12 +29,13 @@ public abstract class Person {
     protected final Set<Tag> tags = new HashSet<>();
     protected final Optional<Name> pairedWithName;
     protected final Optional<Integer> pairedWithId;
+    protected final int timeServed;
 
     /**
      * Every field must be present and not null.
      */
     public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags, Role role,
-                  Optional<Name> pairedWithName, Optional<Integer> pairedWithId) {
+                  Optional<Name> pairedWithName, Optional<Integer> pairedWithId, int timeServed) {
```

Comment:
It's great that you added the timeServed attribute to keep track of the hours for volunteers!